menpo/landmark/test/landmark_test.py

```diff
@@ -118,12 +118,12 @@ def test_LandmarkManager_group_order():
     man = LandmarkManager()
     man['test_set'] = pcloud.copy()
     man['abc'] = pcloud.copy()
-    man['def'] = pcloud.copy()
```

Comment:
Interesting! So this seems to match exactly what we would expect from duck typing onto a class...


<img width="542" alt="screenshot 2017-02-07 09 13 47" src="https://cloud.githubusercontent.com/assets/1312873/22684807/d188e502-ed15-11e6-9ea6-d72f548d14a9.png">